devel/build

```
@@ -28,10 +35,13 @@ docker build \
     --target builder \
     .
 
-docker build \
+docker buildx build \
+    --platform $platform \
```

…and here as well of course.

```suggestion
docker buildx build \
    --builder "$builder" \
    --platform $platform \
```

---

Done: e70bf410aa936a83a5330a89af4570ccb807cf77